packages/client-typescript/docs/guide/methods/send.md

```
@@ -0,0 +1,321 @@
+All three methods require a **token** from a previously started pipeline (via `use()`).
+
+---
+
+## **send()**
+
+Send text or binary data to a running pipeline and get the processing result.
+
+### Method Signature
+
+**Python (async):**
+```python
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)